velox/common/caching/SsdFile.cpp

```diff
@@ -652,12 +671,14 @@ void SsdFile::initializeCheckpoint() {
   std::ifstream state(fileName_ + kCheckpointExtension);
   if (!state.is_open()) {
     hasCheckpoint = false;
+    ++stats_.errorsOpenCheckpoint;
```

Comment:
Is this an error?

Reply:
I guess so. When checkpointIntervalBytes_ > 0, the checkpoint files are supposed to exist.

This error case could happen when the cache files exist while the checkpoint files are wiped out. We could like to signal this case, whether or not we are treating it as error by implementation.

Reply:
```
How about we name it
initCheckpoints? because it could be a first time initialization?
```